server/routes/api-v1/tournamentRoutes.ts

```diff
@@ -0,0 +1,8 @@
+import express from 'express';
+import tournamentController from '../../controllers/tournament/';
+
+const router = express.Router();
+
+router.post('', tournamentController.create);
```

Comment:
router.post('/', tournamentController.create) instead

Reply:
hmm it's working too, so I think `/` is useless here

Reply:
I'd say that `'/'` expresses the intent much more clearly.

Reply:
I agree with You. From visual side its better